README.md

```diff
@@ -86,13 +93,24 @@ You can see the live demo at: https://linkshub.vercel.app
     ```sh
    npm run dev
    ```
-7. Push your changes into your branch
+
+7. Make your changes to the source code.
+
+8. Stage your changes and commit:
+    ```sh
+   git add <filename>
+    ```
+
```

Comment:
Also, we should create a new step for committing the changes

`9. commit your changes`

Reply:
Ok thanks, I will do it.